.github/workflows/script/update_md.sh

```diff
@@ -24,13 +24,19 @@ export KIND_CREATE_CLUSTER_OUTPUT=$( cat create-cluster.log | tee test.log )
 solo init | tee init.log
 export SOLO_INIT_OUTPUT=$( cat init.log | tee test.log )
 
-solo deployment create -i node1,node2,node3 -n "${SOLO_NAMESPACE}" --context kind-${SOLO_CLUSTER_NAME} --email "${SOLO_EMAIL}" --deployment-clusters kind-${SOLO_CLUSTER_NAME} --deployment "${SOLO_DEPLOYMENT}" | tee deployment-create.log
+solo cluster-ref connect --cluster-ref kind-${SOLO_CLUSTER_NAME} --context kind-${SOLO_CLUSTER_NAME} --email "${SOLO_EMAIL}" | tee cluster-ref-connect.log
+export SOLO_CLUSTER_REF_CONNECT_OUTPUT=$( cat cluster-ref-connect.log | tee test.log )
+
+solo deployment create -n "${SOLO_NAMESPACE}" --deployment "${SOLO_DEPLOYMENT}" | tee deployment-create.log
 export SOLO_DEPLOYMENT_CREATE_OUTPUT=$( cat deployment-create.log | tee test.log )
 
+solo deployment add-cluster --deployment "${SOLO_DEPLOYMENT}" --cluster-ref kind-${SOLO_CLUSTER_NAME} --num-consensus-nodes 3 --dns-base-domain cluster.local --dns-consensus-node-pattern network-{nodeAlias}-svc.{namespace}.svc | tee deployment-add-cluster.log
```

Comment:
```suggestion
solo deployment add-cluster --deployment "${SOLO_DEPLOYMENT}" --cluster-ref kind-${SOLO_CLUSTER_NAME} --num-consensus-nodes 3 | tee deployment-add-cluster.log
```
I believe those two flags can be removed since you were setting them to their defaults.